Tests/conf.json

```
@@ -4988,6 +4988,11 @@
             "fromversion": "6.0.0",
             "memory_threshold": 90
         },
+        {
+            "playbookID": "IAMInitOktaUser - Test",
+            "integrations": ["Okta v2","EWSO365"],
+            "fromversion": "6.5.0"
+        },
```

```suggestion
                    "instance_names": ["{enter_instance_name}"]
        },
```
Test fails in the build, choose the right instance to run EWS0365 with 
```
[2022-12-19 15:25:00] - [MainThread] - [ERROR] - There are 2 instances of EWSO365, please select one of them by using the instance_name argument in conf.json. The options are:
ewso365_dev_team
ewso365_sec_eng_team
```